lib/modules/minifyJs.es6

```diff
@@ -53,7 +53,7 @@ function processScriptNode(scriptNode, terserOptions) {
 
     let content = result.code;
     if (isCdataWrapped) {
-        content = '//<![CDATA[' + content + '//]]>';
+        content = '/*<![CDATA[*/' + content + '/*]]>*/';
```

Comment:
Before the PR,

```js
// <![CDATA[
const x = 2;
// ]]>
```

will become

```js
//<![CDATA[const x = 2;//]]>
```

Which the code has been completely commented out.

After the PR

```js
// <![CDATA[
const x = 2;
// ]]>
```

will become

```js
/*<![CDATA[*/const x = 2;/*]]>*/
```

which is the desired behavior.